src/data/wallets/wallet-data.ts

```diff
@@ -1153,7 +1153,7 @@ export const walletsData: WalletData[] = [
     firefox: false,
     chromium: false,
     hardware: true,
-    open_source: false,
+    open_source: true,
```

Comment:
Evidence please.

Reply:
Unless this boolean has a different meaning for hardware companies in this list of software wallets, Ledger Live is open source, as can be seen in our GitHub repo: https://github.com/LedgerHQ/ledger-live/. Or even backend should be open sourced? depending on your criteria.

But honestly, I have no interest in debating it. If you want to set it to false, go ahead